### PR TITLE
init

### PR DESCRIPTION
**Pull request by if0s**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
   },
   "engines": {
-    "node": "14"
+    "node": "16"
   },
   "author": "elastic.io GmbH",
   "license": "Apache-2.0",
   "dependencies": {
-    "@elastic.io/component-commons-library": "2.0.1",
+    "@elastic.io/component-commons-library": "2.1.0-dev.6",
```
Comment (olegosh) [inline package.json:19]: on't forget to change the lib version after testing the component

**Review by olegosh — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -11,14 +11,14 @@
     "url": "git@github.com:elasticio/sftp-component.git"
   },
   "engines": {
-    "node": "14"
+    "node": "16"
```
Comment (EmptyInfinity) [inline package.json:14]:
```suggestion
    "node": "16.x"
```

File: spec/actions/lookupObject.spec.js [hunk 3]
```diff
@@ -39,7 +39,8 @@ describe('SFTP test - lookup file by file name', () => {
       },
     ];
     const sftpClientListStub = sinon.stub(Sftp.prototype, 'list').returns(list);
-    const sftpClientGetStub = sinon.stub(Sftp.prototype, 'get').returns(buffer);
+    // const sftpClientGetStub = sinon.stub(Sftp.prototype, 'get').returns(buffer);
```
Comment (EmptyInfinity) [inline spec/actions/lookupObject.spec.js:42]:
```suggestion
```

**Review by EmptyInfinity — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (if0s) [reply package.json]: done

**Pull request merged**